ocdskingfisherprocess/transform/upgrade_1_0_to_1_1.py

```diff
@@ -72,8 +74,17 @@ def add_status(database, connection):
                 'source_release_id': release_row.id,
             })
 
-        self.store.store_file_item(file_model.filename, None, 'release_package', package, file_item_model.number,
-                                   before_db_transaction_ends_callback=add_status)
```

Comment:
would prefer ` allow_existing_collection_file_item_table_row` to be an argument to `store.store_file_item` as this stops the duplicate code below.

Reply:
I did think about that, but in the end I decided that then the meaning of the store class was getting a bit too fluid and that might cause problems later.

Also, if you mean the duplicate code as in the make package_data stuff that needs to change anyway: #81